docs/operator/helm-getting-started.md

```diff
@@ -0,0 +1,477 @@
+    path: /metrics/cadvisor
+    port: https-metrics
+    relabelings:
+    - sourceLabels:
+      - __metrics_path__
+      targetLabel: metrics_path
+    scheme: https
+    tlsConfig:
+      insecureSkipVerify: true
+  namespaceSelector:
+    matchNames:
+    - default
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: kubelet
+```
+
+These two ServiceMonitors configure Agent to scrape all the kubelet and cAdvisor endpoints in your Kubernetes cluster (one of each per Node). In addition, it defines a `job` label which you may change (it is preset here for compatibility with Grafana Cloud's Kubernetes integration), and allowlists a core set of Kubernetes metrics to reduce remote metrics usage. If you don't need this allowlist, you may omit it, however note that your metrics usage will increase significantly. 
+
+ When you're done, Agent should now be shipping kubelet and cAdvisor metrics to your remote Prometheus endpoint.
+
+## Step 5: Deploy LogsInstance and PodLogs resources
+
+In this step, you'll deploy a LogsInstance resource to collect logs from your cluster nodes and ship these to your remote Loki endpoint. Under the hood, Agent Operator will deploy a DaemonSet of Agents in your cluster that will tail log files defined in PodLogs resources.
+
+Deploy the LogsInstance into your cluster:
+
+```yaml
+apiVersion: monitoring.grafana.com/v1alpha1
+kind: LogsInstance
+metadata:
+  name: primary
+  namespace: default
+  labels:
+    agent: grafana-agent-logs
+spec:
+  clients:
+  - url: your_remote_logs_URL
+    basicAuth:
+      username:
+        name: primary-credentials-logs
+        key: username
+      password:
+        name: primary-credentials-logs
+        key: password
+
+  # Supply an empty namespace selector to look in all namespaces.
+  podLogsNamespaceSelector: {}
+  podLogsSelector:
+    matchLabels:
+      instance: primary
+```
+
+This LogsInstance will pick up PodLogs resources with the `instance: primary` label. Be sure to set the Loki URL to the correct push endpoint (for Grafana Cloud, this will be something like `logs-prod-us-central1.grafana.net/loki/api/v1/push`, however you should check the Cloud Portal to confirm). 
+
+Also note that we are using the `agent: grafana-agent-logs` label here, which will associate this LogsInstance with the GrafanaAgent resource defined in Step 2. This means that it will inherit requests, limits, affinities and other properties defined in the GrafanaAgent custom resource. 
+
+Create the Secret for the LogsInstance resource:
+
+```
+apiVersion: v1
+kind: Secret
+metadata:
+  name: primary-credentials-logs
+  namespace: default
+stringData:
+  username: 'your_username_here'
+  password: 'your_password_here'
+```
+
+If you're using Grafana Cloud, you can find your hosted Loki endpoint username and password in the [Grafana Cloud Portal](https://grafana.com/profile/org ). You may wish to base64-encode these values. In this case, please use `data` instead of `stringData`. 
```

Comment:
```suggestion
If you're using Grafana Cloud, you can find your hosted Loki endpoint username and password in the [Grafana Cloud Portal](https://grafana.com/profile/org). You may wish to base64-encode these values. In this case, please use `data` instead of `stringData`. 
```

Reply:
Note that Kubernetes will automatically base64-encode the Secret when it's sent to the API. If you do a `kubectl get` after applying the secret, you'll see that it's already been encoded; `stringData` just exists as a convenience to the user.

Reply:
Gotcha, will change the language to "base64-encode these values yourself"